ui/components/app/modals/metametrics-opt-in-modal/metametrics-opt-in-modal.component.js

```diff
@@ -27,76 +27,87 @@ export default class MetaMetricsOptInModal extends Component {
               <img src="images/metrics-chart.svg" alt="" />
             </div>
             <div className="metametrics-opt-in__title">
-              Help Us Improve MetaMask
+              {t('metametricsHelpImproveMetaMask')}
             </div>
             <div className="metametrics-opt-in__body">
               <div className="metametrics-opt-in__description">
-                MetaMask would like to gather usage data to better understand
-                how our users interact with the extension. This data will be
-                used to continually improve the usability and user experience of
-                our product and the Ethereum ecosystem.
+                {t('metametricsOptInDescription')}
               </div>
               <div className="metametrics-opt-in__description">
-                MetaMask will..
+                {t('metametricsCommitmentsIntro')}
               </div>
 
               <div className="metametrics-opt-in__committments">
                 <div className="metametrics-opt-in__row">
                   <i className="fa fa-check" />
                   <div className="metametrics-opt-in__row-description">
-                    Always allow you to opt-out via Settings
+                    {t('metametricsCommitmentsAllowOptOut')}
                   </div>
                 </div>
                 <div className="metametrics-opt-in__row">
                   <i className="fa fa-check" />
                   <div className="metametrics-opt-in__row-description">
-                    Send anonymized click & pageview events
+                    {t('metametricsCommitmentsSendAnonymizedEvents')}
                   </div>
                 </div>
                 <div className="metametrics-opt-in__row">
                   <i className="fa fa-check" />
                   <div className="metametrics-opt-in__row-description">
-                    Maintain a public aggregate dashboard to educate the
```

Comment:
Ah, I was wondering what was going on here - I thought we localized this already! We did, but only during onboarding. This second copy of the metametrics agreement was forgotten.

This is intended to match the onboarding metrics agreement exactly, which has had this bullet removed (since we didn't actually deliver on this commitment, and had no immediate plans to). This should be compared with the onboarding page to ensure no other differences exist either.

Reply:
I'm not even entirely sure we need this modal still :thinking: It was used to catch an edge case where a user might bypass the metrics screen. I don't recall how that might occur.

Reply:
I can remove this bullet to match the full page metrics component.

I was able to get the modal to appear like so:

1. Setup MetaMask, select import
2. On the metrics opt-in tab, navigate to `initialize/create-password/import-with-seed-phrase`
3. Enter seed phrase + new password
4. Logout once your on the home screen
5. Log back in
6. See the modal

Reply:
Ah cool, nice job! Probably best to keep it anyway as a safeguard.

Reply:
This bullet has been removed from the modal for consistency